### DISCUSSION TITLE
Django compatibilty update

### DISCUSSION THREAD

**Discussion by hisie** [django-parler/django-parler#346] [Q&A]
Hello, 

I read in the doc "Django versions 2.2 up to 4.0". Django LTS 4.2 is now 8 month old. I have not seen any PR for upgrading django support and the issue about "Is this package still maintained? (https://github.com/django-parler/django-parler/issues/341)" has not any answer from the development team.
This commit (https://github.com/irtazaakram/django-parler/commit/ced8d25da5b90dd352129d4738e2e757a62523c4) from a forked django-parler adds something like django 4.2 support. 

I would like to upgrade one project using django-parler from 3.2 to 4.2, but if this compatibility is not assured, il will do a translation system change instead to avoid risques. 

Can you tell me if there is anybody working actively in this project?

Thank you very much.

**Comment by th3hamm0r** [comment]
Just fyi, I'm using it with Django 5.2 (LTS) and wagtail (through [wagtail-parler](https://github.com/webu/wagtail-parler/)) and I haven't found issues so far.
But yeah, the inactivity is not optimal :neutral_face:
